queries/sxhkdrc/injections.scm

```diff
@@ -1,3 +1 @@
-(command) @bash
-
```

Comment:
Was this on purpose? Just asking since you removed it in the stylua commit

Reply:
Ah, removing it actually had nothing to do with stylua, forgot to split that into its own commit.
Yes it was on purpose, bash injections don't play that nice with sxhkdrc commands